packages/ndla-tabs/src/FilterTabs.tsx

```diff
@@ -108,7 +148,7 @@ class FilterTabs extends Component {
           focusOnSelected: true,
         },
         () => {
-          this.props.onChange(this.props.options[mainTabSelected].value);
+          this.props.onChange(this.props.options[mainTabSelected!].value);
```

Comment:
For å unngå non-null assertion kan man heller legge det inni en `if (mainTabSelected) `

Reply:
Oppdatert.